client/src/components/common/ExportFiles.jsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import { Menu, MenuButton, MenuList, MenuItem, Button } from '@chakra-ui/react';
+
+const ExportFiles = ({ section, data }) => {
+  // const section = {props.section};
+  // const data = {props.data};
+
+  const handleExportPDF = () => {};
+
+  const handleExportCSV = () => {
+    var dateTime = getDateTime();
+    var arrData = data;
+
+    arrData.forEach(function (currentItem) {
```

Comment:
Couldn't `data` be used directly? I don't see `arrData` being modified anywhere, maybe I missed it?

Reply:
I have to delete "_id" field from mongoDB.  The loop at the line below.

Reply:
I see